pkg/network/tracer/tracer_linux_test.go

```diff
@@ -528,8 +528,20 @@ func (s *TracerSuite) TestTranslationBindingRegression() {
 	_, err = c.Write([]byte("ping"))
 	require.NoError(t, err)
 
-	// Give enough time for conntrack cache to be populated
-	time.Sleep(100 * time.Millisecond)
+	// wait for conntrack update
+	laddr := c.LocalAddr().(*net.TCPAddr)
+	raddr := c.RemoteAddr().(*net.TCPAddr)
+	cs := network.ConnectionStats{
+		DPort:  uint16(raddr.Port),
+		Dest:   util.AddressFromNetIP(raddr.IP),
+		Family: network.AFINET,
+		SPort:  uint16(laddr.Port),
+		Source: util.AddressFromNetIP(laddr.IP),
+		Type:   network.TCP,
+	}
+	require.Eventually(t, func() bool {
+		return tr.conntracker.GetTranslationForConn(cs) != nil
+	}, 3*time.Second, 100*time.Millisecond, "timed out waiting for conntrack update")
```

Comment:
nit: can we use a more aggressive retry interval ?  USM has been using smaller intervals and in aggregate it can make a big dent (taking minutes off the   test task)

Reply:
They seem to be also mostly using 100ms ticks.